### ISSUE TITLE
Analityics endpoint

### ISSUE THREAD

**Issue by CucumisSativus** [data-hunters/metadata-digger-server#2]
There should be an endpoint, on url `/api`/v1/analytics`, on GET request it should return a json with system analytics. Currently only number of images in our system will be returned. It should return a valid JSON, with `images_number` field returned as integer example:
```
{
  images_number: 123
}
```

- [ ] Create new package analytics
- [ ] Create Router (you can take a look at PhotosRouter
- [ ] Create analytics service (class, which accepts `PhotosRepository` in constructor)
- [ ] Get the number of photos in system by calling `count` on `PhotosRepository` instance
- [ ] Prepare a DTO, which will be serialized to json see `SearchResponse` usages
- [ ] Create a handler which will accept analytics service, call the proper method there and prepare http response (check photos handler for example)
- [ ] Prepare unit tests for Analytics Service and the route you created

**Comment by NotNor** [comment]
> Get the number of photos in system by calling count on PhotosRepository instance

What about exception handling? Should it return ie. -1?

**Comment by CucumisSativus** [comment]
It's better to return explicit errors. Like
```
{
  "error": "cannot connect to sorl"
}
```

In case of unpredicted error, let it just crash and return 500 internal server error

**Comment by CucumisSativus** [comment]
Resolved by #7
